tools/clang/test/HLSLFileCheck/hlsl/workgraph/ast-rwnodeinput.hlsl

```diff
@@ -46,7 +47,7 @@ void node05(RWDispatchNodeInputRecord<RECORD> input) {}
 [NumThreads(1024,1,1)]
 void node06(RWGroupNodeInputRecords<RECORD> input) {}
 
-//CHECK: ClssTemplateDecl {{0x[0-9a-fA-F]+}} <<invalid sloc>> <invalid sloc> implicit RWGroupNodeInputRecords
+//CHECK: ClassTemplateDecl {{0x[0-9a-fA-F]+}} <<invalid sloc>> <invalid sloc> implicit RWGroupNodeInputRecords
```

Comment:
I wondered how this passed before and I think I know. There should be a bar between `%s` and `FileCheck` in the run line above.

Reply:
I hadn't spotted that! - and as a result the test didn't actually test anything. I have updated this test and related ones to make them more consistent.